pages/desktop/frontend/versions.py

```diff
@@ -23,3 +30,94 @@ def versions_page_header(self):
     def latest_version_number(self):
         el = self.find_element(*self._latest_version_locator).text
         return el.split()[1].replace('Version ', '')
+
+    @property
+    def notice_message(self):
+        return self.find_elements(*self._notice_message_locator)[0]
+
+    @property
+    def notice_message_text(self):
+        text = self.find_elements(*self._notice_message_text_locator)[0].text
+        text += self.find_elements(*self._notice_message_text_locator)[1].text
+        return text
+
+    @property
+    def rating_card(self):
+        el = self.find_element(By.CLASS_NAME, 'AddonSummaryCard')
+        return RatingStats(self, el)
+
+    @property
```

Comment:
Is this necessary? Why would you not access the latest version card directly through the `version_list` object when necessary?